internal/graph/check.go

```diff
@@ -596,6 +601,11 @@ func (c *LocalChecker) ResolveCheck(
 		}, nil
 	}
 
+	typesys, ok := typesystem.TypesystemFromContext(ctx)
+	if !ok {
+		panic("typesystem missing in context")
```

Comment:
Any reason why we would use a panic here? And can we get a small test for this since this is the entry point?

Reply:
No reason. I dislike this panic as much as you do, but let's tackle that in https://github.com/openfga/openfga/issues/616 :)

Reply:
Please add the test 😅 Cuz #616 seems like a large endeavor that might take a while to land.

Reply:
https://github.com/openfga/openfga/pull/1809